bus-mapping/src/circuit_input_builder.rs

```diff
@@ -1524,9 +1524,17 @@ impl<'a> CircuitInputBuilder {
         eth_block: &EthBlock,
         geth_traces: &[eth_types::GethExecTrace],
     ) -> Result<(), Error> {
+        // record tx gas cumulative used
+        let mut tx_cumulative_gas: HashMap<usize, u64> = HashMap::new();
```

Comment:
You don't need to mention the type here, since it's known from the `handle_tx` function call.
```suggestion
        let mut tx_cumulative_gas = HashMap::new();
```

Reply:
seems correct, will have a try !

Reply:
updated in https://github.com/appliedzkp/zkevm-circuits/pull/466/commits/db106a33e264ff347da007fa2028dfc9ee8d0063 thanks